helpers/postgres.rb

```diff
@@ -56,8 +56,19 @@ def postgres_post(name)
     end
   end
 
-  def postgres_list
+  def postgres_list(tag_key: nil, tag_value: nil)
```

Comment:
I think we should accept a list of key/values, so that people can filter by multiple tags. We can accept a single `tags` parameter, which would be an array of key/values. What do you think?